core/keychain/keychain.go

```diff
@@ -222,15 +223,17 @@ func (kc *keychain) DeleteKeypair() error {
 
 	// Note: currently ignoring error on keychain removal because it's failing randomly.
 	// Use GenerateKeyPair with override option instead.
-	ring.Remove(PrivateKeyStoreKey)
+	err = ring.Remove(PrivateKeyStoreKey)
+	if err != nil {
+		log.Error("Error removing keychaing from keyring", err)
```

Comment:
Maybe this should be a `log.Warn()` instead.

Reply:
This is a weird one.  It's actually an error scenario hence the `Error` but we are just ignoring it temporarily because it happens intermittently. Maybe it's something that's fixed cc @dmerrill6. Either way, I'm removing the whole `if` block in the meantime to leave it like how it was.